Removed external dependencies from public lib

onursumer opened the pull request:
# What? Why?
Fix cBioPortal/cbioportal/issues/6376.

# Checks
- [ ] Is this PR adding logic based on one or more **clinical** attributes? If yes, please make sure validation for this attribute is also present in the data validation / data loading layers (in backend repo) and documented in [File-Formats Clinical data section](https://github.com/cBioPortal/cbioportal/blob/master/docs/File-Formats.md#clinical-data)!
- [ ] Follows [7 rules of great commit messages](http://chris.beams.io/posts/git-commit/). For most PRs a single commit should suffice, in some cases multiple topical commits can be useful. During review it is ok to see tiny commits (e.g. Fix reviewer comments), but right before the code gets merged to master or rc branch, any such commits should be squashed since they are useless to the other developers. Definitely avoid [merge commits, use rebase instead.](http://nathanleclaire.com/blog/2014/09/14/dont-be-scared-of-git-rebase/)
- [ ] Follows the [Airbnb React/JSX Style guide](https://github.com/airbnb/javascript/tree/master/react).
- [ ] Make sure your commit messages end with a Signed-off-by string (this line
  can be automatically added by git if you run the `git-commit` command with
  the `-s` option)

onursumer commented:
@inodb  @alisman  I think we should make `yarn buildPublicLib` part of CI so that we won't merge breaking changes in the future.

onursumer commented:
We decided to hardcode the font URL to avoid this confusing workaround

inodb commented:
@onursumer 	could you maybe at it to CI as part of this PR? It can be added as a new job here: https://github.com/cBioPortal/cbioportal-frontend/blob/master/.circleci/config.yml#L65. Maybe call it something like `build_public_lib_test` or something?

onursumer commented:
@alisman I've simplified the PR by making the font url hardcoded.

@ino I've added the new `public_lib` test into circle CI config. It is not marked as `Required` though, I think we need to change that on GitHub settings after merging this PR, right?

inodb commented on webpack.config.js:
```diff
@@ -127,7 +127,6 @@ var config = {
             styles: join(src, 'styles'),
             pages: join(src, 'pages'),
             shared: join(src, 'shared'),
-            commons: join(src, 'commons'),
```
what's this?
